hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/TestStorageContainerManager.java

```diff
@@ -779,7 +707,7 @@ public void testScmProcessDatanodeHeartbeat() throws Exception {
     conf.setClass(NET_TOPOLOGY_NODE_SWITCH_MAPPING_IMPL_KEY,
         StaticMapping.class, DNSToSwitchMapping.class);
     StaticMapping.addNodeToRack(NetUtils.normalizeHostNames(
-        Collections.singleton(HddsUtils.getHostName(conf))).get(0),
+            Collections.singleton(HddsUtils.getHostName(conf))).get(0),
         "/rack1");
```

Comment:
Now I noticed this can be simplified by using `normalizeHostName(String)` instead of `normalizeHostNames(Collection)`:

```
StaticMapping.addNodeToRack(NetUtils.normalizeHostName(HddsUtils.getHostName(conf)),
    "/rack1");
```

Note: `import ...Collections` becomes unused.